.github/workflows/main.yml

```diff
@@ -11,6 +11,10 @@ env:
   REPO: linode/apl-core
   DOCKER_PASSWORD: ${{ secrets.DOCKERHUB_LINODEBOT_TOKEN }}
   DOCKER_USERNAME: ${{ vars.DOCKERHUB_LINODEBOT_USERNAME }}
+  DEV_KUBECONFIG_64: ${{ secrets.DEV_KUBECONFIG }}
+  GIT_USER: svcAPLBot
```

Comment:
In GH Action variables there is already "BOT_USERNAME " variable. Perhaps you can leverage that one instead of creating yet another variable - GIT_USER